packages/atlas/src/views/viewer/PlaceBid/PlaceBid.tsx

```diff
@@ -0,0 +1,332 @@
+import React, { useEffect, useState } from 'react'
+import { CSSTransition } from 'react-transition-group'
+
+import { Avatar } from '@/components/Avatar'
+import { DrawerHeader } from '@/components/DrawerHeader'
+import { Information } from '@/components/Information'
+import { NftCard } from '@/components/NftCard'
+import { Pill } from '@/components/Pill'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { SvgAlertsWarning24 } from '@/components/_icons'
+import { SvgJoystreamLogoShort } from '@/components/_illustrations'
+import { TextField } from '@/components/_inputs/TextField'
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+import { useCountdown } from '@/hooks/useCountdown'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { useJoystream, useTokenPrice } from '@/providers/joystream'
+import { usePlaceBid } from '@/providers/placeBid/hooks'
+import { useUser } from '@/providers/user'
+import { cVar } from '@/styles'
+import { formatDurationShort } from '@/utils/time'
+
+import {
+  ActiveBidWrapper,
+  BidderName,
+  Container,
+  Content,
+  CurrentBidAvatar,
+  CurrentBidJoyToken,
+  CurrentBidWrapper,
+  Divider,
+  EndingTime,
+  FlexWrapper,
+  Header,
+  InnerContainer,
+  Messages,
+  MinimumBid,
+  MinimumBidWrapper,
+  NftPreview,
+  PaymentSplitValues,
+  PaymentSplitWrapper,
+  PlaceBidWrapper,
+  Row,
+  StyledActionBar,
+  StyledJoyTokenIcon,
+  Timer,
+} from './PlaceBid.styles'
+
+const DUMMY_NFT_TILE_PROPS = {
+  role: 'owner' as const,
+  auction: 'none' as const,
+  thumbnail: { thumbnailUrl: 'https://placedog.net/360/203' },
+  creator: { assetUrl: 'https://placedog.net/100/100?random=1', name: 'Jane' },
+  owner: { assetUrl: 'https://placedog.net/100/100?random=2', name: 'Kate' },
+  loading: false,
+  fullWidth: false,
+}
+
+const TRANSACTION_FEE = 19
+const FIXED_PRICE = 500
+const MINIMUM_BID = 301
+const TIME_LEFT = 80
+const BID = {
+  bidder: {
+    name: 'Mike Shipa',
+  },
+  amount: 341,
+}
+
+export const PlaceBid = () => {
```

Comment:
```suggestion
export const PlaceBid: React.FC = () => {
```
I mean, I don't agree with this rule to adding React.FC everywhere, I think we shouldn't use React.FC at all, but Klaudiusz will definitely point this out 😄

Reply:
Also, I think it should be named differently. maybe we could name this AuctionView like in in figma?
![image](https://user-images.githubusercontent.com/51168865/156775997-f2ec5604-79ae-44e1-902c-ccd522eb5042.png)

Reply:
`AuctionView` I would say `AuctionView` is a bit ambiguous, `PlaceBid` is more explicit

Reply:
> I think we shouldn't use React.FC at all

@drillprop Why?

Reply:
There are several reasons for that, for example, React.FC always implies `children`. I think we had a short discussion about it before. This article sums it up nicely.
https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/

Reply:
Ah yes, remember that. If we rewrite the codebase at some point, let's revisit, but for now let's continue with what we have :D